protobuf/lib/src/protobuf/extension_field_set.dart

```diff
@@ -172,10 +172,10 @@ class _ExtensionFieldSet {
     _isReadOnly = true;
     for (var field in _info.values) {
       if (field.isRepeated) {
-        final entries = _values[field.tagNumber];
+        final entries = _values[field.tagNumber] as PbList<GeneratedMessage>?;
```

Comment:
The failure is related to this change which made the cast to strong. Notice that previously it would only cast to `PbList<GeneratedMessage>` if `field.isGroupOrMessage`. 

I think the right way is to cast to `PbList<Object>` here.

Reply:
Ah ha, thanks!